### DISCUSSION TITLE
Replacing DocsPage With `.stories.mdx`

### DISCUSSION THREAD

**Discussion by iamkkwak** [storybookjs/storybook#15716] [Help]
Hello,

I'm doing this: https://storybook.js.org/docs/vue/writing-docs/docs-page#with-mdx-documentation on Vue,
But seems like it is only compatible with `.mdx` not `.stories.mdx`.

I tried to import `.stories.mdx`, but it says
`
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.
`

Should I convert `.stories.mdx` to `string` type? How?
or Is there another way to replace *only* docs page?

Any solutions for this?
Thanks in advance for any help! :)

**Comment by garridio85** [comment]
Did you ever get this working? Having the hardest time upgrading an existing storybook to use mdx
